spec/helpers/project_helpers.rb

```diff
@@ -33,8 +33,9 @@ def project_id
       query[query_re, 1].to_i
     end
 
-    def create_project (path = '/search/collections?p=!C179003030-ORNL_DAAC!C1214558039-NOAA_NCEI', name='Test Project')
-      user = User.first
+    # def create_project (path = '/search/collections?p=!C179003030-ORNL_DAAC!C1214558039-NOAA_NCEI', name='Test Project')
```

Comment:
Old def still remains in comment - best to delete

Reply:
Yes because when the bug is fixed we should restore this